pkg/typedef/index.go

```diff
@@ -0,0 +1,23 @@
+// Copyright 2019 ScyllaDB
```

Comment:
There is no point to have separate file just of one struct, please move it back to `typedef.go`

Reply:
Done